frb_rust/src/misc/logging.rs

```diff
@@ -0,0 +1,233 @@
+#[macro_export]
+macro_rules! enable_frb_logging {
+    () => {
+#[allow(clippy::crate_in_macro_def)]
+use crate::frb_generated::StreamSink;
+use flutter_rust_bridge::frb as frb_4_log;
+pub use log::{LevelFilter, Metadata, Record};
+
+#[frb(dart_code = "
+import 'dart:io';
+import 'package:logging/logging.dart';
+
+static void default_log_function(Log2DartLogRecord record) {
+  print(
+    '[${DateTime.now()} ${log_level_from_number(record.levelNumber)} @${record.rustLog ? 'Rust' : 'Dart'} ${record.loggerName}] ${record.message}');
+}
+
+/// initialize the logging system, including the rust logger
+static Logger init_logger(
```

Comment:
nit: IIRC we should have camelCase in Dart

Reply:
yes, I did this now everywhere in the user visible functions.

I did not do that in internal (_bla_bla) functions within the macro ... I think it is better readble this way - but I have not strong opinion on that, can do it there as well.

Reply:
> I think it is better readble this way

I guess following Dart's convention will be a bit more readable

Reply:
Ultimately I have to agree :) It affected only one call, `_log_level_from_str`, which is now `_logLevelFromStr`.

I actually mixed it up with `initialize_log2dart`, which is a rust function, that I changed to initialize_log_2_dart`.